### PR TITLE
document how to override the language negotiation/redirect

### PR DESCRIPTION
**Pull request by erral**
Following my answer on community.plone.org to a question regarding the redirect that Plone does when configuring plone.app.multilingual and several languages, I have brought that piece of documentation here.

### CODE REVIEW

File: docs/i18n-l10n/language-negotiation.md [hunk 1]
```diff
@@ -91,8 +91,17 @@ This language switcher will only rely on the user preferred language to decide w
...
 -   They may write their own view, and configure the ZMI.
     To configure the ZMI, visit `www.domain.com/portal_types/Plone%20Site/manage_propertiesForm` or navigate there as an Admin user, {guilabel}`username > Site Setup`, {guilabel}`Advanced > Management Interface`, {guilabel}`portal_types`, and finally {guilabel}`Plone Site`.
     Set the value of `Default view method` to the name of the view.
+
+
+## Changing the default behavior
+
+If for any reason you want to change the default behavior set when using `plone.app.multilingual`, you have 2 options:
+
+1- Override the `language-switcher` view. Plone has a view called `language-switcher` defined in [plone.app.multilingual.browser.switcher.py](https://github.com/plone/plone.app.multilingual/blob/master/src/plone/app/multilingual/browser/switcher.py) which handles the redirection from the root of the Plone site to the proper Language Root Folder. You can override this view using the usual techniques to provide your own implementation.
```
Comment (jensens) [inline docs/i18n-l10n/language-negotiation.md]: Numbered list Syntax wrong
```suggestion
1. Override the `language-switcher` view. Plone has a view called `language-switcher` defined in [plone.app.multilingual.browser.switcher.py](https://github.com/plone/plone.app.multilingual/blob/master/src/plone/app/multilingual/browser/switcher.py) which handles the redirection from the root of the Plone site to the proper Language Root Folder. 
   You can override this view using the usual techniques to provide your own implementation.
```

File: docs/i18n-l10n/language-negotiation.md [hunk 2]
```diff
@@ -91,8 +91,17 @@ This language switcher will only rely on the user preferred language to decide w
...
+
+2- Create a new view: you can create your own view with your own implementation, configure it properly and then set it as a `default view` for `Plone Site` objects. To do so, you may need to provide your own installation profile with a `Plone Site.xml` file and with the proper configuration.
```
Comment (jensens) [inline docs/i18n-l10n/language-negotiation.md]:
```suggestion
2. Create a new view: you can create your own view with your own implementation, configure it properly and then set it as a `default view` for `Plone Site` objects. 
   To do so, you may need to provide your own installation profile with a `Plone Site.xml` file and with the proper configuration.
```

File: docs/i18n-l10n/language-negotiation-volto.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+In its configuration, the component tries to match the `I18N_LANGUAGE` cookie set in the user's browser with the list of supported languages, and if the match does not succeed, it selects the default language configured in Volto.
+
+After that it does the redirection to the matched Language Root Folder.
+
+If the site is not configured to be multilingual, Volto doesn't do any redirect.
+
+## Overriding the default behavior
+
+To do so, you need to provide your own `MultilingualRedirector` component [customizing it](https://6.docs.plone.org/volto/recipes/customizing-components.html).
```
Comment (jensens) [inline docs/i18n-l10n/language-negotiation-volto.md]: Internal references must not be absolute URLs.

**Review by jensens — CHANGES_REQUESTED** [review]

File: docs/i18n-l10n/language-negotiation-volto.md (hunk 3 above)
Comment (stevepiercy) [reply docs/i18n-l10n/language-negotiation-volto.md]: Fixed in 5fc218edbd23ac527fb17ff60c58110bd2ed8673

File: docs/i18n-l10n/language-negotiation.md (hunk 1 above)
Comment (stevepiercy) [reply docs/i18n-l10n/language-negotiation.md]: Fixed this and the next in f223cf9e820a2ec450c04243e9cd5afd5eebead1 (along with several other mistakes).

File: docs/i18n-l10n/language-negotiation.md (hunk 2 above)
Comment (stevepiercy) [reply docs/i18n-l10n/language-negotiation.md]: Fixed in f223cf9e820a2ec450c04243e9cd5afd5eebead1

**Comment by stevepiercy** [comment]
The failing workflow can be ignored. I have a fix in https://github.com/plone/documentation/pull/1542

**Review by stevepiercy — DISMISSED** [review]
LGTM. Thank you!

**Pull request merged**